examples/usecases/retrieval-with-hyperparameter-optimization.ipynb

```diff
@@ -0,0 +1,1405 @@
+{
```

Comment:
<p>may be remove <code>among</code> from  this <code>how many out of top 100 retrieved candidates were among purchased items</code>  sentence? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/786/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
good point, removed!